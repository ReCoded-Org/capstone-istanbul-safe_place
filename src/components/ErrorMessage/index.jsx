import React from 'react'
import './index.scss';
import PatchExclamationFll from '../../images/Patch-exclamation-fll.svg'

export default function ErrorMessage({ message = 'Invalid input value' }) {
    return (
        <div className="errorMessage" role="alert">
            <img src={PatchExclamationFll} alt="" width="32" height="32" title="Bootstrap" />
            <p>
                {message}
            </p>
        </div>
    )
}
