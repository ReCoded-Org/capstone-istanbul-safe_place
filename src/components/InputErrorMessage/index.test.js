import React from "react";
import renderer from "react-test-renderer";
import ErrorMessage from "./index";

it("renders correctly and matches the snapshot", () => {
  const tree = renderer.create(<ErrorMessage />).toJSON();
  expect(tree).toMatchSnapshot();
});
