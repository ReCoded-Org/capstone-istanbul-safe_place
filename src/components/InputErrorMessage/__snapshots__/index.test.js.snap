// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly and matches the snapshot 1`] = `
<div
  className="errorMessage"
  role="alert"
>
  <img
    alt="Error Logo"
    height="32"
    src="patchExclamationFill.svg"
    width="32"
  />
  <p />
</div>
`;
